lib/govuk_index/presenters/elasticsearch_presenter.rb

```diff
@@ -49,6 +49,64 @@ def document
+        continuation_link:                  specialist.continuation_link,
+        industries:                         specialist.industries,
+        types_of_support:                   specialist.types_of_support,
+        will_continue_on:                   specialist.will_continue_on,
+        opened_date:                        specialist.opened_date,
+        closed_date:                        specialist.closed_date,
+        case_type:                          specialist.case_type,
+        case_state:                         specialist.case_state,
+        market_sector:                      specialist.market_sector,
+        outcome_type:                       specialist.outcome_type,
+        grant_type:                         specialist.grant_type,
+        land_use:                           specialist.land_use,
+        tiers_or_standalone_items:          specialist.tiers_or_standalone_items,
+        funding_amount:                     specialist.funding_amount,
+        dfid_document_type:                 specialist.dfid_document_type,
+        country:                            specialist.country,
+        first_published_at:                 specialist.first_published_at,
+        dfid_authors:                       specialist.dfid_authors,
+        dfid_theme:                         specialist.dfid_theme,
+        dfid_review_status:                 specialist.dfid_review_status,
+        tribunal_decision_country:          specialist.tribunal_decision_country,
+        fund_state:                         specialist.fund_state,
+        fund_type:                          specialist.fund_type,
+        funding_source:                     specialist.funding_source,
+        closing_date:                       specialist.closing_date,
+        development_sector:                 specialist.development_sector,
+        eligible_entities:                  specialist.eligible_entities,
+        value_of_funding:                   specialist.value_of_funding,
+        vessel_type:                        specialist.vessel_type,
+        alert_type:                         specialist.alert_type,
+        issued_date:                        specialist.issued_date,
+        medical_specialism:                 specialist.medical_specialism,
+        railway_type:                       specialist.railway_type,
+        fault_type:                         specialist.fault_type,
+        faulty_item_type:                   specialist.faulty_item_type,
+        alert_issue_date:                   specialist.alert_issue_date,
+        manufacturer:                       specialist.manufacturer,
+        faulty_item_model:                  specialist.faulty_item_model,
+        serial_number:                      specialist.serial_number,
+        build_start_date:                   specialist.build_start_date,
+        build_end_date:                     specialist.build_end_date,
```

Comment:
Are we going to send all of these fields even when the document is not a specialist document? I know we send nil for a few fields at the moment but that's a lot of extra fields which a high percentage of documents won't be using. It feels like we should be splitting common fields, expanded links, specialist etc out.

Reply:
Why.. 

Sending everything might make the code slightly slower, but the alternative is to have more complex code. 

Do we perceive speed or complexity to be the most important? It will be running on threaded background workers.

Reply:
NOTE: I am happy to change this so we limit the fields if we believe that gives us an advantage, it just for me the biggest advantage would be from simplified code which this approach gives us. 

The other advantage of only using fields specified in the logic is that we can ensure the validity of our data, but again I don't think that is a strong enough argument given that the publishing app, schemas, and publishing API should already be doing this.